src/layouts/components/Footer.tsx

```diff
@@ -25,8 +34,45 @@ export function Footer (): JSX.Element {
               <FooterExternalLink url='https://dex.defichain.com/mainnet/anchors' text='BTC Anchors' />
             </div>
           </div>
+          <div className='flex-1'>
+            <div className='text-2xl font-semibold'>Social</div>
+            <div className='flex flex-wrap mt-3  w-84'>
+              <div className='flex flex-row space-x-2 items-center w-1/2'>
+                <FaTwitterSquare size={25} />
```

Comment:
```suggestion
                <FaTwitterSquare size={24} />
```

Always try to use even numbers due to down/up scale algorithm where odd multiple won't look as favorable due to 2x, 3x screens